build.gradle

```diff
@@ -4,6 +4,10 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '6.1.0'
 }
 
+apply plugin: 'java'
```

Comment:
it should still work without this as we declare java a plugin in the `plugins {}` stanza - and this is an older version of declaring plugins